src/saving/SaveHelper.cs

```diff
@@ -131,11 +131,89 @@ private static Save CreateSaveObject(MainGameState gameState, SaveInformation.Sa
         };
     }
 
+    private static string BuildQuicksaveFilename(uint quicksaveId)
```

Comment:
I've selected in other code that `quicksave` is not one word, so you need to edit a bunch of places to be `QuickSave` as it is two separate words.

Reply:
Also `uint` shouldn't be used here, it seems that in C# uint isn't really used, so change that to int.